app/test_main.py

```diff
@@ -1,3 +1,43 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+
+    @pytest.mark.parametrize("cat_age, dog_age, expected", [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (100, 100, [21, 17]),
```

Comment:
The test cases for valid ages are well-covered according to the task description. However, consider adding edge cases such as negative numbers or very large values to ensure the function handles these scenarios correctly as per the checklist requirement.